src/tasks/installers/installer.ts

```diff
@@ -80,9 +80,8 @@ export class InstallerTasks {
     } else if (flags.installer === 'operator') {
       title = '🏃‍  Running the Eclipse Che operator'
       task = () => {
-        // The operator installs Eclipse Che in multiuser mode only
+        // The operator installs Eclipse Che in multiuser mode by default
```

Comment:
Maybe, make sense to have a flag single user, to have ability fast deploy Che in this mode without playing with CR yaml or CR yaml patch?

Reply:
Single-user mode for operator is what I would like to hide.
It is just to deprecate `deploy.sh` script